src/pivotal-ui/components/iconography/iconography.scss

```diff
@@ -13,47 +13,87 @@ categories:
-<i class="fa fa-download"></i>                     | `.fa.fa-download`      | Renders our download icon
+Icon                                                    | Module and Filename
+------------------------------------------------------- | ----------------------------------------------------
+<svg class="icon"><use xlink:href="#i-download"/></svg> | `@npmcorp/glyphish/svg/g4/331-download.svg`
+<svg class="icon"><use xlink:href="#i-plus"/></svg>     | `@npmcorp/glyphish/svg/g4/436-plus.svg`
+<svg class="icon"><use xlink:href="#i-warn"/></svg>     | `@npmcorp/glyphish/svg/g5/582-warning-sign.svg`
+<svg class="icon"><use xlink:href="#i-mbox"/></svg>     | `@npmcorp/glyphish/svg/g6/Normal/805-mailbox@2x.svg`
+<svg class="icon"><use xlink:href="#i-keyhole"/></svg>  | `@npmcorp/glyphish/svg/g6/Normal/720-keyhole@2x.svg`
+<svg class="icon"><use xlink:href="#i-package"/></svg>  | `@npmcorp/glyphish/svg/g6/Normal/807-package@2x.svg`
+<svg class="icon"><use xlink:href="#i-users"/></svg>    | `@npmcorp/glyphish/svg/g6/Normal/895-user-group@2x.svg`
+<svg class="icon"><use xlink:href="#i-bdfl"/></svg>     | `@npmcorp/glyphish/svg/g8/Normal/1225-basket-delete@2x.svg`
+
+Icons must be defined inline once per page.
+
+The icon defined:
+
+```html_example
+<svg style="display: none">
+ <symbol id="i-download" width="23.979" height="21.964" viewBox="0 0 23.979 21.964">
+   <path d="M15.992 17.966zm-5.89-15.453c.638.27 1.2.668 1.7 1.09.503.428.88.955 1.207 1.457.3.523.545 1.034.703 1.533.177.49.258.965.3 1.382.056.433.047.76.042 1.018.006.112-.005.277-.016.424l-.024.36-.014.21h-4l5.988 7.98 5.99-7.98h-3.984v-.02l-.008-.07-.036-.297c-.01-.097-.025-.216-.042-.356-.016-.136-.033-.294-.082-.543a10.674 10.674 0 0 0-.375-1.572 11 11 0 0 0-.77-1.854A11.116 11.116 0 0 0 15.37 3.31c-.556-.63-1.193-1.25-1.95-1.763-.76-.503-1.603-.943-2.52-1.205a8.69 8.69 0 0 0-2.832-.33c-.302.02-.403.03-.695.075-.26.034-.49.09-.717.143a8.23 8.23 0 0 0-1.312.468 8.61 8.61 0 0 0-2.17 1.47A10.158 10.158 0 0 0 .69 5.85a9.737 9.737 0 0 0-.674 2.99 7.72 7.72 0 0 0 .006 1.147l.01.093a.496.496 0 0 0 .988-.093s-.027-.363.03-1.01c.068-.636.21-1.585.708-2.614.494-1.014 1.25-2.16 2.37-3.01a6.938 6.938 0 0 1 1.85-1.016 6.46 6.46 0 0 1 1.026-.262c.175-.022.35-.047.504-.052.117-.015.435-.015.546-.006a6.55 6.55 0 0 1 2.047.5zM7.98 18.963v3h16v-3h-16z"/>
+  </symbol>
+</svg>
+This box should not show the icon, since it is just a symbol definition
+```
+
+The icon instantiated:
 
 ```html_example
-<i class="fa fa-download type-brand-8"></i>
+<svg class="icon"><use xlink:href="#i-download"/></svg>
+```
+
+Using the [handlebars icon helper](https://www.npmjs.com/package/handlebars-helper-icon) when developing with handlebars, you can have it inline the icon for you like below, and it will make sure that the first instance includes the definition, so only this one helper call is required.
+
+```hbs
+{{icon "@npmcorp/glyphish/svg/g4/331-download" class="icon type-brand-8"}}
```

Comment:
What sort of custom attributes did you mean?

This is passed through literally, so it integrates using CSS classes like any other component -- the `type-brand-8` is an existing class.